randovania/games/samus_returns/exporter/patch_data_factory.py

```diff
@@ -104,6 +104,7 @@ def _calculate_starting_inventory(self, resources: ResourceCollection) -> dict[s
                 print(f"Skipping {resource} for starting inventory: no item id")
                 continue
         result["ITEM_MAX_LIFE"] = self.configuration.starting_energy
+        result["ITEM_MAX_SPECIAL_ENERGY"] = result.pop("ITEM_MAX_SPECIAL_ENERGY", 0) + self.configuration.starting_aeion
```

Comment:
This line is weird.
I guess if you start with multiple aeion abilities, additional aeion needs to be summed up in the loop before instead of just assign it (because that would always be the last value)

Reply:
Figured that some other part already sums it up